p1pop.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="s1pop.css" type="text/css" rel="stylesheet">
+        <link href="css/fontello.css" type="text/css" rel="stylesheet">
+
+        <meta charset="Utf-8">
+    </head>
+    <body>
+        <main>
+               <div class="navbar">
+                <i class="demo-icon icon-search"></i>                
+                <ul>
+                    <li>About me</li>
```

Comment:
Szkoda że nie dodałeś tagu `<a>` dla elementów nawigacji aby podać ścieżki do konkretnych `section` lub konkretnych podstron. Teraz jest to sam text.